CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 # CHANGELOG
 
-## unreleased
+## v0.1.14 (beta) - Apr 26 2018
```

Comment:
```suggestion
## v0.1.14 (beta) - Apr 26th 2019
```

Reply:
Good catch on the year.  I assumed you were just tagging the `th`.

Reply:
I was initially but luckily spotted it from the corner of my eye. :)

Reply:
Welp, I'm stuck in the past.